website/project/model.py

```diff
@@ -2992,8 +3011,15 @@ class Sanction(StoredObject):
     REJECTION_NOT_AUTHORIZED_MESSAEGE = 'This user is not authorized to reject this {DISPLAY_NAME}'
     REJECTION_INVALID_TOKEN_MESSAGE = 'Invalid rejection token provided for this {DISPLAY_NAME}.'
 
-    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+    # Controls whether or not the Sanction needs unanimous approval or just a single approval
+    ANY = 'any'
+    UNANIMOUS = 'unanimous'
```

Comment:
@abought this is consistent with how we've worked out `Sanction` and relevant subclasses. I like the way it's done (highest level of class it applies too) but it is something worth thinking about and putting into the style guides.